src/main/java/eu/xenit/gradle/docker/compose/DockerComposeConventionImpl.java

```diff
@@ -108,23 +118,33 @@ public void fromProject(Project project) {
                 composeTask.doFirst(CONFIGURE_COMPOSE_ACTION_NAME, new Action<Task>() { // No lambda -> see #96
                     @Override
                     public void execute(Task t) {
-                        createSetComposeEnvironmentFromPathAction().execute(dockerBuildImage);
+                        if(environmentVariable != null) {
+                            createSetComposeEnvironmentAction(environmentVariable).execute(dockerBuildImage);
```

Comment:
@vierbergenlars Hier moet ik het volgende doen? 

`createSetComposeEnvironmentAction(environmentVariable) + t.getName()).execute(dockerBuildImage);`

Of is dit fout?

Reply:
The code as currently exists is only correct for the case where only one task of type `DockerBuildImage` exists in the project. Else, the environment variable would be overwritten.

You can indeed to `createSetComposeEnvironmentAction(environmentVariable+"_"+t.getName()).execute(dockerBuildImage)`.
Don't forget to add the underscore, else you get environment variables that are mashed together strangely.